jupyter-lib/lib-ext/src/main/kotlin/org/jetbrains/kotlinx/jupyter/ext/graph/wrappers/HClassLoader.kt

```diff
@@ -0,0 +1,18 @@
+package org.jetbrains.kotlinx.jupyter.ext.graph.wrappers
+
+import org.jetbrains.kotlinx.jupyter.ext.graph.NodeWrapper
+import java.net.URLClassLoader
+
+class HClassLoader(node: ClassLoader) : NodeWrapper<ClassLoader>(node) {
```

Comment:
Not clear what H means. It is also better to create an extension function like `inline fun <reified T> GraphNode.Companion.fromClassLoader<T>(): GraphNode` for builders. It would greatly improve discoverability.

Reply:
Good point, thanks!